### DISCUSSION TITLE
How to do log files when running in daemon mode in Mac OS ?

### DISCUSSION THREAD

**Discussion by Badsheep** [mikebrady/shairport-sync#1755] [Q&A]
The question is in the title... 
I have some strange issues that would like to debug... thanks !

**Comment by mikebrady** [comment]
Well, I'd suggest setting the `log_verbosity` to `1` in the configuration file. Not sure where the logs will come out in macOS, but you should be able to find them with the Console app.

**Reply by Badsheep** [reply]
Thanks !
